config/themePluginConfig.ts

```diff
@@ -0,0 +1,115 @@
+export default {
+  theme: [
+    {
+      key: 'dark',
+      fileName: 'dark.css',
+      theme: 'dark',
+    },
+    {
+      key: 'dust',
+      fileName: 'dust.css',
+      modifyVars: {
+        '@primary-color': '#F5222D',
+      },
+    },
+    {
+      key: 'volcano',
+      fileName: 'volcano.css',
+      modifyVars: {
+        '@primary-color': '#FA541C',
+      },
+    },
+    {
+      key: 'sunset',
+      fileName: 'sunset.css',
+      modifyVars: {
+        '@primary-color': '#FAAD14',
+      },
+    },
+    {
+      key: 'cyan',
+      fileName: 'cyan.css',
+      modifyVars: {
+        '@primary-color': '#13C2C2',
+      },
+    },
+    {
+      key: 'green',
+      fileName: 'green.css',
+      modifyVars: {
+        '@primary-color': '#52C41A',
+      },
+    },
+    {
+      key: 'geekblue',
+      fileName: 'geekblue.css',
+      modifyVars: {
+        '@primary-color': '#2F54EB',
+      },
+    },
+    {
+      key: 'purple',
+      fileName: 'purple.css',
+      modifyVars: {
+        '@primary-color': '#722ED1',
+      },
+    },
+
+    {
+      key: 'dust',
+      theme: 'dark',
+      fileName: 'dark-dust.css',
+      modifyVars: {
+        '@primary-color': '#F5222D',
+      },
+    },
+    {
+      key: 'volcano',
+      theme: 'dark',
+      fileName: 'dark-volcano.css',
+      modifyVars: {
+        '@primary-color': '#FA541C',
+      },
+    },
+    {
+      key: 'sunset',
+      theme: 'dark',
+      fileName: 'dark-sunset.css',
+      modifyVars: {
+        '@primary-color': '#FAAD14',
+      },
+    },
+    {
+      key: 'cyan',
+      theme: 'dark',
+      fileName: 'dark-cyan.css',
+      modifyVars: {
+        '@primary-color': '#13C2C2',
+      },
+    },
+    {
+      key: 'green',
+      theme: 'dark',
+      fileName: 'dark-green.css',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'fileName' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=3667353)